src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -1065,13 +1067,18 @@ internal bool HasIsByRefLikeAttribute(EntityHandle token)
             if (info.HasValue)
             {
                 ObsoleteAttributeData obsoleteData = TryExtractObsoleteDataFromAttribute(info);
-                if (obsoleteData != null &&
-                    ignoreByRefLikeMarker &&
-                    obsoleteData.Message == ByRefLikeMarker)
+                if (obsoleteData == null)
```

Comment:
Consider moving this is to a switch statement, so future additions are one-line changes.